<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="labAddPeriod">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>0</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(4, 0, 62);
color:rgb(255, 255, 255);
font: 15pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>تسجيل فترة  جديدة</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeAttendanceTime">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>hh:mm AP</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::LocalTime</enum>
   </property>
   <property name="time">
    <time>
     <hour>3</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>157</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>ميعاد الانصراف</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeLeavingTime">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>158</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>hh:mm AP</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
   <property name="time">
    <time>
     <hour>2</hour>
     <minute>0</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>ميعاد الحضور</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="timeAllowedTimeForLeaving">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>8</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>سماحية الحضور</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="timeAllowedTimeForAttendance">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>8</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>سماحية الانصراف</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtPricePeriod">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>195</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Motken K Sina</family>
     <pointsize>8</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color:rgb(255, 255, 255);
    font: 75 8pt &quot;Motken K Sina&quot;;
    color:rgb(0, 0, 0);
    border-radius:10px;
	padding: 2px 0;
	border: 1px solid #5b586e;

}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditorInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>195</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>سعر الفترة</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAddingperiod">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>إلغــــــاء</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/exclamation-48.ico</normaloff>:/icons_qrc/exclamation-48.ico</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveperiod">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>103</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton {
background-color:rgb(50, 65, 75);
 
 color:rgb(255, 255, 255);
font: 75 12pt &quot;Motken K Sina&quot;;

  border-radius:10px;
	padding: 10px 5;
	
}

QPushButton:hover {
background-color:rgb(191, 194, 181);
      	 color:rgb(0, 0, 0);
		font: 75 10pt &quot;Motken K Sina&quot;;

       

}</string>
   </property>
   <property name="text">
    <string>حفـــظ</string>
   </property>
   <property name="icon">
    <iconset resource="icons_rc/icons.qrc">
     <normaloff>:/icons_qrc/true.png</normaloff>:/icons_qrc/true.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Motken K Sina&quot;;</string>
   </property>
   <property name="text">
    <string>اسم الفترة</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combPeriodName">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255);
color:rgb(0, 0, 0);
font: 10pt &quot;Motken K Sina&quot;;
    border-radius:10px;
	
	border: 1px solid #5b586e;

</string>
   </property>
   <item>
    <property name="text">
     <string>الفترة الاولى</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>الفترة الثانية</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="icons_rc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
